From a90506e4ed81751b615b0cc235488e2bb7d8c645 Mon Sep 17 00:00:00 2001
From: jizhenlo <zhenlong.z.ji@intel.com>
Date: Wed, 16 Nov 2022 16:58:46 +0800
Subject: [PATCH] Enforce the debugfs restriction

Google requires vendors to remove debugfs from their product devices
from android 11. And from android 12, Google added some cts test
cases to check this.

Tracked-On: OAM-104016
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>
---
 groups/device-specific/caas/caas.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/groups/device-specific/caas/caas.mk b/groups/device-specific/caas/caas.mk
index da0ab0c..95d1102 100644
--- a/groups/device-specific/caas/caas.mk
+++ b/groups/device-specific/caas/caas.mk
@@ -65,7 +65,11 @@ PRODUCT_VENDOR_PROPERTIES += \
     ro.soc.manufacturer=$(PRODUCT_MANUFACTURER)
 
 PRODUCT_RESTRICT_VENDOR_FILES := false
+ifneq ($(TARGET_BUILD_VARIANT),user)
 PRODUCT_SET_DEBUGFS_RESTRICTIONS := false
+else
+PRODUCT_SET_DEBUGFS_RESTRICTIONS := true
+endif
 {{^ota-update}}
 PRODUCT_SHIPPING_API_LEVEL := 31
 PRODUCT_OTA_ENFORCE_VINTF_KERNEL_REQUIREMENTS := true
-- 
2.38.1

